docker-for-aws/persistent-data-volumes.md

```diff
@@ -8,7 +8,17 @@ title: Docker for AWS persistent data volumes
 
 ## What is Cloudstor?
 
-Cloudstor is a volume plugin managed by Docker. It comes pre-installed and pre-configured in swarms deployed on Docker for AWS. Swarm tasks use a volume created through Cloudstor to mount a persistent data volume that stays attached to the swarm tasks no matter which swarm node they get scheduled or migrated to. Cloudstor relies on shared storage infrastructure provided by AWS to allow swarm tasks to create/mount their persistent volumes on any node in the swarm. In a future release we will introduce support for direct attached storage to satisfy very low latency/high IOPs requirements.
+Cloudstor is a modern volume plugin managed by Docker. It comes pre-installed and pre-configured in Docker Swarms deployed through Docker for AWS. Docker Swarm tasks as well as regular containers can use a volume created through Cloudstor to mount a persistent data volume. In Docker for AWS, Cloudstor can be used with two `backing` options: `local` (that uses EBS) and `shared` (that uses EFS) described below.
```

Comment:
I agree with your point and that's the reason we have various defaults in place so that commands may work across clouds with minimal changes. 

One additional point to consider is if we do want to implement s3 support down the line, we will probably need something other than `shared` to differentiate EFS from s3 although s3 backing will offer `shared` ish semantics.

One possible set of names could be `direct-attached`/`network-attached` which aligns with the usual storage terminology to distinguish SCSI/ATA from NFS/SMB. For s3 we can probably make something up along the lines of `object-attached`. Thoughts?